dvc/repo/status.py

```diff
@@ -91,22 +91,26 @@ def _cloud_status(
     )
 
     ret = {}
-    status_info = self.cloud.status(
-        used_objs, jobs, remote=remote, log_missing=False
-    )
-    for info in status_info.values():
-        name = info["name"]
-        status_ = info["status"]
-        if status_ == cloud.STATUS_OK:
+    for odb, objs in used.items():
+        if odb is not None:
+            # ignore imported objects
```

Comment:
As discussed, we may want to start reporting status for imports later, but for now we can just skip any objects that come from non-default (None) odbs